### ISSUE TITLE
Skip link to help users bypass the menu on each page

### ISSUE THREAD

**Issue by MelanieTPX** [tpximpact/mhclg-oruk#90]
According to WCAG criteria 2.4.1 Bypass Blocks (Level A), I think this is saying that there should be a way for keyboard only users to skip directly to the main content area on each page? 

https://www.w3.org/WAI/WCAG22/quickref/?showtechniques=133%2C134%2C135%2C141%2C144%2C145%2C1410%2C1411%2C1412%2C211%2C212%2C242%2C243%2C246%2C258%2C241#bypass-blocks

I'm not too sure on whether we have this already, so happy to take advice on whether this is needed for accessibility. When I tabbed through webpages on keyboard I couldn't find a way to skip through all the menu links every time, even when the menu wasn't expanded.

![Image](https://github.com/user-attachments/assets/57b7208a-80f1-4e13-b2cf-2568dfd5076e)

**Comment by gjhltn** [comment]
We use this approach instead. 

https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/ARIA11
